packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -77,10 +77,6 @@ const tabs: VerticalTabItemProps[] = [
         name: "general",
         href: "/settings/organizations/general",
       },
-      {
-        name: "members",
```

Comment:
can you make the same change in `SettingsLayoutAppDirClient`? We currently keep two same layouts until we fully migrate to app router